packages/react-dev-utils/ModuleScopePlugin.js

```diff
@@ -49,7 +49,7 @@ class ModuleScopePlugin {
         )
       );
       // Error if in a parent directory of src/
-      if (requestRelative[0] === '.') {
+      if (requestRelative.startsWith('../')) {
```

Comment:
Thank you for the windows "reminder"!
I also fixed the first detection (different situation though).